docs/notes.md

```diff
@@ -191,6 +191,124 @@ Content-Type: application/vnd.api+json
 }
 ```
 
+### GET /notes
+
+It returns the list of notes, sorted by last update.
+
+**Note:** a permission on `GET io.cozy.files` is required to use this route.
+
+#### Request
+
+```http
+GET /notes HTTP/1.1
+Host: alice.example.net
+Accept: application/vnd.api+json
+```
+#### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.api+json
+```
+
+```json
+{
+  "links": {
+    "next": "/notes?page[cursor]=a078d6f0-04a9-0138-3e03-543d7eb8149c"
+  },
+  "data": [{
+    "type": "io.cozy.files",
+    "id": "bf0dbdb0-e1ed-0137-8548-543d7eb8149c",
+    "meta": {
+      "rev": "1-f71ee54e2"
+    },
+    "attributes": {
+      "type": "file",
+      "name": "My new note.cozy-note",
+      "trashed": false,
+      "md5sum": "NjhiMzI5ZGE5ODkzZTM0MDk5YzdkOGFkNWNiOWM5NDAgIC0K",
+      "created_at": "2019-11-05T12:38:04Z",
+      "updated_at": "2019-11-05T12:38:04Z",
+      "tags": [],
+      "metadata": {
+        "title": "My new note",
+        "content": { "type": "doc", "content": [{ "type": "paragraph" }] },
+        "version": 0,
+        "schema": {
+          "nodes": [
+            ["doc", { "content": "block+" }],
+            ["paragraph", { "content": "inline*", "group": "block" }],
+            ["blockquote", { "content": "block+", "group": "block" }],
+            ["horizontal_rule", { "group": "block" }],
+            [
+              "heading",
+              {
+                "content": "inline*",
+                "group": "block",
+                "attrs": { "level": { "default": 1 } }
+              }
+            ],
+            ["code_block", { "content": "text*", "marks": "", "group": "block" }],
+            ["text", { "group": "inline" }],
+            [
+              "image",
+              {
+                "group": "inline",
+                "inline": true,
+                "attrs": { "alt": {}, "src": {}, "title": {} }
+              }
+            ],
+            ["hard_break", { "group": "inline", "inline": true }],
+            [
+              "ordered_list",
+              {
+                "content": "list_item+",
+                "group": "block",
+                "attrs": { "order": { "default": 1 } }
+              }
+            ],
+            ["bullet_list", { "content": "list_item+", "group": "block" }],
+            ["list_item", { "content": "paragraph block*" }]
+          ],
+          "marks": [
+            ["link", { "attrs": { "href": {}, "title": {} }, "inclusive": false }],
+            ["em", {}],
+            ["strong", {}],
+            ["code", {}]
+          ],
+          "topNode": "doc"
+        }
+      },
+      "size": 1,
+      "executable": false,
+      "class": "text",
```

Comment:
Should we create a `note` class? Today we are looking at the extension and a few metadata (schema / title / version / content) to detect if the file is a note. Having a more robust solution can be a good point :)

Reply:
It will be the `mime` that should be used to detect notes. And I need to update the docs!